.github/files/gh-autorelease/files/autorelease.sh

```diff
@@ -2,32 +2,39 @@
 
 set -eo pipefail
 
+: "${GH_TOKEN:?Build argument needs to be set and non-empty.}"
 : "${GITHUB_REF:?Build argument needs to be set and non-empty.}"
 : "${GITHUB_SHA:?Build argument needs to be set and non-empty.}"
-: "${API_TOKEN_GITHUB:?Build argument needs to be set and non-empty.}"
-: "${GITHUB_API_URL:?Build argument needs to be set and non-empty.}"
-: "${GITHUB_REPOSITORY:?Build argument needs to be set and non-empty.}"
+
+if [[ ! -f composer.json ]]; then
+	echo '::error::No composer.json. Did it get excluded from the mirror?'
+	exit 1
+fi
 
 ## Determine tag
-if [[ ! "$GITHUB_REF" =~ ^refs/tags/v?[0-9]+(\.[0-9]+)+(-[a-z0-9._-]+)?$ ]]; then
-	echo "::error::Expected GITHUB_REF like \`refs/tags/v1.2.3\` or \`refs/tags/1.2.3\`, got \`$GITHUB_REF\`"
+ROLLING_MODE=
+if [[ "$GITHUB_REF" =~ ^refs/tags/v?[0-9]+(\.[0-9]+)+(-[a-z0-9._-]+)?$ ]]; then
+	TAG="${GITHUB_REF#refs/tags/}"
+elif [[ "$GITHUB_REF" == "refs/heads/trunk" ]]; then
+	ROLLING_MODE="$(jq -r '.extra.autorelease.rolling-release' composer.json)"
+	if [[ -z "$ROLLING_MODE" ]]; then
+		echo "::notice::Skipping trunk release because autorelease rolling mode is not enabled."
+		exit 0
+	fi
```

Comment:
This isn't going to work right, for several reasons:

1. `jq` syntax is incorrect.
2. After you fix that, it'll probably still blow up when `.extra.autorelease` is set to true rather than an object.
3. When it's unset, you'll get the string `null` rather than the empty string.

What I'd do is more like this:
```suggestion
	if ! jq -e '.extra.autorelease["rolling-release"]? // false' composer.json > /dev/null; then
		echo "::notice::Skipping trunk release because autorelease rolling mode is not enabled."
		exit 0
	fi
	ROLLING_MODE=true
```

<details><summary>What that does</summary>

* `jq -e` has jq's exit code indicate whether the last thing it output was truthy or falsey.
* Since we don't care about the output, just the exit code, `> /dev/null` to suppress the output.
* For an object key that contains non-identifier characters like `-`, we have to do like `["rolling-release"]` to access it.
* The `?` after `["rolling-release"]` tells it not to error out if `["rolling-release"]` can't be accessed because `.extra.autorelease` isn't an object.
* The `// false` is probably unnecessary other than to make the behavior more obvious to humans if the value is unset.

</details>